src/main/java/com/dougnoel/sentinel/steps/TableVerificationSteps.java

```diff
@@ -85,20 +85,29 @@ public static void verifyUniqueColumnText(String columnName, String isMultiCells
 	}
 
     /**
-     * Verifies the given table contains the given column
+     * Verifies the given table contains or does not contain the given column
      * <p>
      * <b>Gherkin Examples:</b>
      * <ul>
      * <li>I verify the Contact Us table contains the Phone Number column </li>
      * <li>I verify the Vision benefits table contains a Deductible column</li>
      * <li>I verify the Discography table contains the Album Name column </li>
+     * <li>I verify the Discography table does not contain the Album Name column </li>
      * </ul>
      * @param tableName String name of the table containing the column
      * @param columnName String name of the column to verify
      */
-    @Then("^I verify the (.*?) contains (?:a|the) (.*?) column$")
-    public static void verifyColumnExists(String tableName, String columnName) {
-        assertTrue(getElementAsTable(tableName).verifyColumnExists(columnName));
+    @Then("^I verify the (.*?)( does not)? contains? (?:a|the) (.*?) column$")
```

Comment:
Let's modify this to do a has or a contains like other text steps. has is an equal, contains is a contains.